hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -153,8 +158,9 @@ def compat_kw(*args, **kw):
             lambda b: b[-1:] != b"\0"
         )
     elif dtype.kind == "u":
-        kw = compat_kw(min_value=0, max_value=2 ** (8 * dtype.itemsize) - 1)
-        result = st.integers(**kw)
+        result = st.integers(
+            **compat_kw(min_value=0, max_value=2 ** (8 * dtype.itemsize) - 1)
+        )
```

Comment:
Try avoid trivial changes like this that go either way

Reply:
Yeah, for consistency `kw` should be `kws` (it's multiple keywords), so I figured removing the name would remove the misnomer and produce a simple piece of code 🤷.  I can also factor out such changes into separate PRs in the future 👍. Sorry.

Reply:
I think this just isn't worth changing, we use `kw` as a shorter form of `kwargs` in many places throughout the repo.  If it *was* worth changing, I'd put it in a separate commit but the same PR.

Reply:
I'll remove it.